### ISSUE TITLE
数据集划分

### ISSUE THREAD

**Issue by xjwwjx** [shawnliang420/AerialMind#5]
这份数据集好像没有指明数据集中哪些是训练集，虽然可以通过seqmap.txt反推出来测试集。

**Comment by xjwwjx** [comment]
> Sorry, bro! My apologies for the oversight! I should have clearly defined the splits in the documentation.
> 
> The Training Set is indeed defined as shown in seqmap.txt. For the Test Set, I have categorized them into In-Domain and Cross-Domain scenarios.
> 
> Test Set Partition In-Domain (VisDrone): `video_ids = ["uav0000009_03358_v", "uav0000073_00600_v", "uav0000073_04464_v", "uav0000077_00720_v", "uav0000088_00290_v", "uav0000119_02301_v", "uav0000120_04775_v", "uav0000161_00000_v", "uav0000188_00000_v", "uav0000201_00000_v", "uav0000249_00001_v", "uav0000249_02688_v", "uav0000297_00000_v", "uav0000297_02761_v", "uav0000306_00230_v", "uav0000355_00001_v", "uav0000370_00001_v"]`
> 
> Cross-Domain(UAVDT): `video_ids = ["M0203", "M0205", "M0209", "M0403", "M0701", "M0801", "M1001", "M1004", "M1007", "M1101", "M1301", "M1302", "M1401"]`
> 
> You can update the if **name** == '**main**': block in the inference.py(e.g, TransRMOT) as follows:
> 
> ```
> if __name__ == '__main__':
>     torch.multiprocessing.set_start_method('spawn')
>     parser = argparse.ArgumentParser('DETR training and evaluation script', parents=[get_args_parser()])
>     args = parser.parse_args()
>     
>     if args.output_dir:
>         Path(args.output_dir).mkdir(parents=True, exist_ok=True)
> 
>     expressions_root = os.path.join(args.rmot_path, 'expression')
>     
>     if "KITTI" in args.rmot_path: 
>         video_ids = ['0005', '0011', '0013', '0019']
>     else: 
>         # Add your target test set here (e.g., In-Domain)
>         video_ids =  ["uav0000009_03358_v", "uav0000073_00600_v", "uav0000073_04464_v","uav0000077_00720_v", "uav0000088_00290_v", "uav0000119_02301_v", "uav0000120_04775_v", "uav0000161_00000_v", "uav0000188_00000_v", "uav0000201_00000_v", "uav0000249_00001_v", "uav0000249_02688_v", "uav0000297_00000_v", "uav0000297_02761_v", "uav0000306_00230_v", "uav0000355_00001_v", "uav0000370_00001_v"]
>         #video_ids =  ["M0203", "M0205", "M0209","M0403", "M0701", "M0801", "M1001", "M1004", "M1007", "M1101", "M1301", "M1302", "M1401"]
> ```

这个数据集是将VisDrone分成两部分，然后UAVDT作为补充的（跨域的）测试集吗？

**Comment by shawnliang420** [comment]
That's right
